app/views/portal/show.rb

```diff
@@ -3,420 +3,432 @@ class Show < ApplicationView
-              },
-              {
-                title: 'site.object.meta-label.publication-date',
-                fields: ['proxies.dctermsPublished']
-              },
-              {
-                title: 'site.object.meta-label.issued',
-                fields: ['proxies.dctermsIssued']
-              },
-              {
-                title: 'site.object.meta-label.temporal',
-                fields: ['proxies.dctermsTemporal']
-              },
+        navigation = {
+          back_url: back_link_url,
+          next_prev: {}
+        }
+        if @previous_document
+          navigation[:next_prev].merge!(
+            prev_url: document_path(@previous_document, format: 'html'),
+            prev_link_attrs: [
               {
-                title: 'site.object.meta-label.place-time',
-                fields: ['proxies.dcCoverage']
+                name: 'data-context-href',
+                value: track_document_path(@previous_document, session_tracking_path_opts(search_session['counter'].to_i - 1))
               }
             ]
-          ),
-          description: data_section(
-            title: 'site.object.meta-label.description',
-            sections: [
-              {
-                title: false,
-                collected: render_document_show_field_value(document, 'proxies.dctermsTOC')
-              },
+          )
+        end
+        if @next_document
+          navigation[:next_prev].merge!(
+            next_url: document_path(@next_document, format: 'html'),
+            next_link_attrs: [
               {
-                title: false,
-                collected: render_document_show_field_value(document, 'proxies.dcDescription')
+                name: 'data-context-href',
+                value: track_document_path(@next_document, session_tracking_path_opts(search_session['counter'].to_i + 1))
               }
             ]
-          ),
-          # download: content_object_download,
-          media: media_items,
-          meta_additional: {
-            geo: {
-              latitude: '"' + (render_document_show_field_value(document, 'places.latitude') || '') + '"',
-              longitude: '"' + (render_document_show_field_value(document, 'places.longitude') || '') + '"',
-              long_and_lat: long_and_lat?,
-              placeName: render_document_show_field_value(document, 'places.prefLabel'),
-              labels: {
-                longitude: t('site.object.meta-label.longitude') + ':',
-                latitude: t('site.object.meta-label.latitude') + ':',
-                map: t('site.object.meta-label.map'),
-                points: {
-                  n: t('site.object.points.north'),
-                  s: t('site.object.points.south'),
-                  e: t('site.object.points.east'),
-                  w: t('site.object.points.west')
+          )
+        end
+        navigation.reverse_merge(helpers.navigation)
+      end
+    end
+
+    def content
```

Comment:
Assignment Branch Condition size for content is too high. [108.1/15]<br>Method has too many lines. [351/10]